examples/window-app/common/window-app.matter

```diff
@@ -1810,6 +1810,91 @@ cluster UserLabel = 65 {
   readonly attribute int16u clusterRevision = 65533;
 }
 
+/** Allows servers to ensure that listed clients are notified when a server is available for communication. */
+cluster IcdManagement = 70 {
```

Comment:
Is this adding of the ICD management server to the window app related to adding CSL to the silabs platform?

Any chance you could update the summary to also des-abbreviate `CSL` and `SSED` ? what do these stand for?

Reply:
Updated the description. 
Enabling the SIT ICD feature set isn't related to the SSED feature set, but enabling the SIT ICD feature allows us to show case both together which is fundamentally the end goal.